cmd/exchange_template/exchange_template_test.go

```diff
@@ -65,18 +66,14 @@ func TestNewExchangeAndSaveConfig(t *testing.T) {
 			WS:   true,
 		},
 	)
-	if err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, err)
 
 	cfgData, err := os.ReadFile(exchangeConfigPath)
-	if err != nil {
-		t.Fatal(err)
-	}
-	if err = saveConfig(testExchangeDir, cfg, exchCfg); err != nil {
-		t.Error(err)
-	}
-	if err = os.WriteFile(exchangeConfigPath, cfgData, file.DefaultPermissionOctal); err != nil {
-		t.Error(err)
-	}
+	require.NoError(t, err)
+
+	err = saveConfig(testExchangeDir, cfg, exchCfg)
+	assert.NoError(t, err)
+
+	err = os.WriteFile(exchangeConfigPath, cfgData, file.DefaultPermissionOctal)
+	assert.NoError(t, err)
```

Comment:
🚧 Should have an assertion description, since this call isn't the subject of a unit test.
🦍 The switch to `require` isn't required, because there's nothing after it. But might as well be consistent with all errors now, I'm thinking.
```suggestion
	require.NoError(t, err, "os.WriteFile must not error")
```